bootstrap.js

```diff
@@ -250,4 +252,20 @@ function xhr(aStr, aOptions={}) {
 	
 	return deferredMain_xhr.promise;
 }
-// end - common helper functions
\ No newline at end of file
+// end - common helper functions
+
+function getCookie(){
+	if(Components.classes["@mozilla.org/cookiemanager;1"]
+		.getService(Components.interfaces.nsICookieManager2)
+		.cookieExists({
```

Comment:
`Services.cookies.cookieExists` didnt work here? Its more perofmance to use Services module.